src/actions/filesActions.js

```diff
@@ -4,10 +4,12 @@ export const UPDATE_A_FILE_SUCCESS = "UPDATE_A_FILE_SUCCESS"
 export const DELETE_A_FILE_SUCCESS = "DELETE_A_FILE_SUCCESS"
 export const RETRIEVE_DIR_SUCCESS = "RETRIEVE_DIR_SUCCESS"
 
-export const makeFetchFilesSuccess = response => ({
-  type: FETCH_FILES_SUCCESS,
-  response
-})
+export const makeFetchFilesSuccess = response => [
```

Comment:
why do you : 
- modify this file
- return an arrray ?? actions are always objects